repos/system_upgrade/common/actors/distributionsignedrpmscanner/tests/test_distributionsignedrpmscanner.py

```diff
@@ -57,12 +60,56 @@ def test_actor_execution_with_signed_unsigned_data(current_actor_context):
+    assert current_actor_context.consume(DistributionSignedRPM)
+    assert len(current_actor_context.consume(DistributionSignedRPM)[0].items) == 5
     assert current_actor_context.consume(InstalledRedHatSignedRPM)
     assert len(current_actor_context.consume(InstalledRedHatSignedRPM)[0].items) == 5
     assert current_actor_context.consume(InstalledUnsignedRPM)
     assert len(current_actor_context.consume(InstalledUnsignedRPM)[0].items) == 4
 
 
+def test_actor_execution_with_signed_unsigned_data_centos(current_actor_context):
+    CENTOS_PACKAGER = 'CentOS BuildSystem <http://bugs.centos.org>'
+    config = mock_configs.CONFIG
+
+    config.os_release = OSRelease(
+        release_id='centos',
+        name='CentOS Linux',
+        pretty_name='CentOS Linux 7 (Core)',
+        version='7 (Core)',
+        version_id='7'
+    )
+
+    installed_rpm = [
+        RPM(name='sample01', version='0.1', release='1.sm01', epoch='1', packager=CENTOS_PACKAGER, arch='noarch',
+            pgpsig='RSA/SHA256, Mon 01 Jan 1970 00:00:00 AM -03, Key ID 24c6a8a7f4a80eb5'),
+        RPM(name='sample02', version='0.1', release='1.sm01', epoch='1', packager=CENTOS_PACKAGER, arch='noarch',
+            pgpsig='SOME_OTHER_SIG_X'),
+        RPM(name='sample03', version='0.1', release='1.sm01', epoch='1', packager=CENTOS_PACKAGER, arch='noarch',
+            pgpsig='RSA/SHA256, Mon 01 Jan 1970 00:00:00 AM -03, Key ID 05b555b38483c65d'),
+        RPM(name='sample04', version='0.1', release='1.sm01', epoch='1', packager=CENTOS_PACKAGER, arch='noarch',
+            pgpsig='SOME_OTHER_SIG_X'),
+        RPM(name='sample05', version='0.1', release='1.sm01', epoch='1', packager=CENTOS_PACKAGER, arch='noarch',
+            pgpsig='RSA/SHA256, Mon 01 Jan 1970 00:00:00 AM -03, Key ID 4eb84e71f2ee9d55'),
+        RPM(name='sample06', version='0.1', release='1.sm01', epoch='1', packager=CENTOS_PACKAGER, arch='noarch',
+            pgpsig='SOME_OTHER_SIG_X'),
+        RPM(name='sample07', version='0.1', release='1.sm01', epoch='1', packager=CENTOS_PACKAGER, arch='noarch',
+            pgpsig='RSA/SHA256, Mon 01 Jan 1970 00:00:00 AM -03, Key ID fd372689897da07a'),
+        RPM(name='sample08', version='0.1', release='1.sm01', epoch='1', packager=CENTOS_PACKAGER, arch='noarch',
+            pgpsig='SOME_OTHER_SIG_X'),
+        RPM(name='sample09', version='0.1', release='1.sm01', epoch='1', packager=CENTOS_PACKAGER, arch='noarch',
+            pgpsig='RSA/SHA256, Mon 01 Jan 1970 00:00:00 AM -03, Key ID 45689c882fa658e0')]
+
+    current_actor_context.feed(InstalledRPM(items=installed_rpm))
+    current_actor_context.run(config_model=config)
+    assert current_actor_context.consume(DistributionSignedRPM)
+    assert len(current_actor_context.consume(DistributionSignedRPM)[0].items) == 3
+    assert current_actor_context.consume(InstalledRedHatSignedRPM)
+    # assert len(current_actor_context.consume(InstalledRedHatSignedRPM)[0].items) == 0
```

Comment:
this assert obviously fails right now, but that's what you wanted to have, right?